src/libraries/sendtohelixhelp.proj

```diff
@@ -62,6 +62,17 @@
     <HelixType Condition="'$(BuildAllConfigurations)' == 'true'">test/functional/packaging/</HelixType>
   </PropertyGroup>
 
+  <PropertyGroup Condition="'$(TargetOS)' == 'Browser'">
+    <IncludeDotNetCli>true</IncludeDotNetCli>
+    <DotNetCliPackageType>sdk</DotNetCliPackageType>
+    <DotNetCliVersion>3.1.201</DotNetCliVersion>
+  </PropertyGroup>
+  <ItemGroup Condition="'$(TargetOS)' == 'Browser'">
+    <HelixPreCommand Include="${HELIX_CORRELATION_PAYLOAD}/dotnet/dotnet tool install --tool-path ${HELIX_WORKITEM_PAYLOAD}/xharness-cli --version $(MicrosoftDotNetXHarnessCLIVersion) --add-source https://dnceng.pkgs.visualstudio.com/public/_packaging/dotnet-eng/nuget/v3/index.json Microsoft.DotNet.XHarness.CLI" />
+    <HelixPreCommand Include="export DOTNET_ROOT=${HELIX_CORRELATION_PAYLOAD}/dotnet" />
+    <HelixPreCommand Include="export PATH=${HELIX_WORKITEM_PAYLOAD}/xharness-cli:$PATH" />
```

Comment:
This could now theoretically be replaced with setting a property
```xml
    <IncludeXHarnessCli>true</IncludeXHarnessCli>
```

And the section with `IncludeDotNetCli` can either be removed completely, or just the `IncludeDotNetCli` prop has to stay.

And once we improve the Helix SDK to pre-install the .NET runtime only and pack the tool as correlation payload, it will happen automatically here

Reply:
We can do this when we're not using Docker to run, which we can do when v8 is available on a non-docker queue

Reply:
Can’t we just fix the docker image to work with the default HelixSDK switches? It is handy to get a repro environment when tests are run in a docker image. That way I can just run them in a container using the same image locally and I get the same behavior